hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -693,13 +693,12 @@ public OmMultipartUploadListParts listParts(String volumeName,
     boolean isTruncated = false;
     int nextPartNumberMarker = 0;
     BucketLayout bucketLayout = BucketLayout.DEFAULT;
-    if (ozoneManager != null) {
```

Comment:
@smengcl I'm fine with creating a cleanup PR against master and then backporting, but @prashantpogde seemed like he was in a bit of a hurry to get this done.  

If so, then it might be better just to remove the unused constructors here and deal with any merge conflicts when we do the merge to master.  (It shouldn't be too bad.)  Either way is fine with me.  Just let me know.

Reply:
> If so, then it might be better just to remove the unused constructors here and deal with any merge conflicts when we do the merge to master. (It shouldn't be too bad.)

Alright, let's do this on the snapshot branch then. We might have to deal with some conflicts if other PRs are changing `OzoneManager` around at the same time but yeah it shouldn't been too bad (most likely just `import` conflicts).